src/Maestrano/Configuration/WebhookConnec.cs

```diff
@@ -6,34 +6,28 @@ namespace Maestrano.Configuration
 {
     public class WebhookConnec
     {
-        public WebhookConnecSubscriptions Subscriptions { get; set; }
-
         /// <summary>
         /// Load Webhook into a WebhookAccount configuration object from a JObject 
         /// </summary>
         /// <returns>A WebhookAccount configuration object</returns>
         public static WebhookConnec LoadFromJson(JObject obj)
         {
             var config = new WebhookConnec();
-            config.NotificationsPath = obj["notification_path"].Value<string>();
-            var subscriptions = obj["connec_subscriptions"];
-            if (subscriptions != null)
-            {
-                config.Subscriptions = WebhookConnecSubscriptions.LoadFromJson(subscriptions.Value<JObject>());
-            }
-            else
-            {
-                config.Subscriptions = new WebhookConnecSubscriptions();
```

Comment:
I would just keep this line for now and flag it as deprecated (blank initialization)

Also I would:
- flag the WebhookConnecSubscriptions as deprecated
- modify all getters in WebhookConnecSubscriptions to return true (subscriptions list are managed in the devplatform anyway so this local subscription should only become of passthrough.

The above will ensure backward compatibility.